.env.test

```diff
@@ -1,5 +1,6 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
-APP_SECRET='s$cretf0rt3st'
+APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
-DATABASE_URL=''
+PANTHER_APP_ENV=panther
+PANTHER_ERROR_SCREENSHOT_DIR=./var/error-screenshots
```

Comment:
Why Panther and not cypress we use in other places?

Reply:
Wait we don't have an UI for imports backend...

Reply:
Updated all files normally installed by flex. Decided to leave defaults in place.